src/util/bip32.rs

```diff
@@ -615,6 +630,20 @@ impl ExtendedPubKey {
         }
     }
 
+    /// Constructs ECDSA compressed public key matching internal public key representation.
+    pub fn to_ecdsa(&self) -> ecdsa::PublicKey {
+        ecdsa::PublicKey {
+            compressed: true,
+            key: self.public_key
+        }
+    }
+
+    /// Constructs BIP340 xcoord-only public key for Schnorr signatures and Taproot use matching
```

Comment:
Let's stick with the term `X-only` instead of `x-coord-only` to be more consistent with bip340

Reply:
Last time I read it it was "X-coord only", but anyway, changed. Omnia mutantur...